ipams/remote/remote.go

```diff
@@ -93,6 +93,8 @@ func (a *allocator) RequestAddress(poolID string, address net.IP, options map[st
 	}
 	if res.Address != "" {
 		retAddress, err = types.ParseCIDR(res.Address)
+	} else {
+		return nil, nil, ipamapi.ErrNoAvailableIPs
```

Comment:
Ah ok, if it's for misbehaving driver. Given the driver is not following the protocol, let's return an error indicating that, rather than treating it as a "no available IPs" one.